trunner/tools/phoenix.py

```diff
@@ -22,6 +22,23 @@ def wait_for_dev(port, timeout=0):
             raise TimeoutError
 
 
+def wait_for_vid_pid(port_hint: str, timeout=0):
```

Comment:
I would renaming it to be more descriptive of its function.
Maybe something like: `find_device_by_vid_pid`